### ISSUE TITLE
Incompatible with Keycloak 21+

### ISSUE THREAD

**Issue by slominskir** [slominskir/KeycloakConditionalSpnegoAuthenticator#2]
It appears this custom authenticator does not work with Keycloak 21+ due to RestEasy lib incompatibility:

```
ERROR [org.keycloak.services.error.KeycloakErrorHandler] (executor-thread-13) Uncaught server error: java.lang.NoSuchMethodError: 'org.jboss.resteasy.spi.HttpRequest org.keycloak.authentication.AuthenticationFlowContext.getHttpRequest()'
        at org.keycloak.extras.authentication.authenticators.browser.ConditionalSpnegoAuthenticator.authenticate(ConditionalSpnegoAuthenticator.java:29)
```

**Comment by slominskir** [comment]
Compare: 

[v20.0.3 getHttpRequest()](https://www.keycloak.org/docs-api/20.0.3/javadocs/org/keycloak/authentication/AbstractAuthenticationFlowContext.html#getHttpRequest())

vs

[v21.0.1 getHttpRequest()](https://www.keycloak.org/docs-api/21.0.1/javadocs/org/keycloak/authentication/AbstractAuthenticationFlowContext.html#getHttpRequest())

**Comment by slominskir** [comment]
Fixed in release [v1.2.0](https://github.com/slominskir/KeycloakConditionalSpnegoAuthenticator/releases/tag/v1.2.0)
